2-mandatory/3-groceries.js

```diff
@@ -28,13 +28,28 @@ Exercise 1:
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
 
+for (const key in weeklyMealPlan) {
+  weeklyMealPlan[key].map((item) => {
+    if (weeklyGroceriesToBuy.includes(item) === false)
+      weeklyGroceriesToBuy.push(item);
+  });
+}
+
 /*
 Exercise 2:
   Loop through your list again, but now only collect the weekend items into the weekendGroceriesToBuy array.
 */
 // Gather weekend item names into this array
 let weekendGroceriesToBuy = [];
 
+for (const key in weeklyMealPlan) {
+  if (key === "saturday" || key === "sunday") {
+    weeklyMealPlan[key].map((item) => {
+      if (weekendGroceriesToBuy.includes(item) === false)
```

Comment:
Normally checking if something is true is done without the `=== true` and checking if something is false is done like `if (!weekendGroceriesToBuy.includes(item))`